pipelines/pipeline_katello_nightly_split.yml

```diff
@@ -0,0 +1,116 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-nightly-split
+    forklift_boxes:
+      pipeline-app-nightly-centos7-split:
+        box: centos7
+        memory: 4096
+        ansible:
+          group: "{{ forklift_name }}"
+      pipeline-services-nightly-centos7-split:
+        box: centos7
+        memory: 4096
+        ansible:
+          group: "{{ forklift_name }}"
+      pipeline-proxy-nightly-centos7-split:
+        box: centos7
+        memory: 3072
+        ansible:
+          group: "{{ forklift_name }}"
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-app-nightly-centos7-split
+  - pipeline-proxy-nightly-centos7-split
+  - pipeline-services-nightly-centos7-split
+  become: yes
+  vars:
+    katello_repositories_version: nightly
+    foreman_repositories_version: nightly
+    katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+    foreman_installer_additional_packages:
+      - foreman-installer-katello
+    foreman_installer_module_prs: 'katello/katello/209,katello/foreman_proxy_content/136'
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - disable_firewall
+    - haveged
+    - update_os_packages
+    - foreman_installer
+
+- hosts: pipeline-services-nightly-centos7-split
+  become: yes
+  vars:
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--no-enable-foreman"
+      - "--no-enable-foreman-cli"
+      - "--no-enable-foreman-plugin-tasks"
+      - "--no-enable-foreman-proxy"
+      - "--no-enable-foreman-proxy-content"
+      - "--no-enable-foreman-proxy-plugin-pulp"
+      - "--no-enable-puppet"
+      - "--katello-manage-application false"
+      - "--katello-manage-pulp true"
+      - "--katello-manage-candlepin true"
+      - "--katello-manage-qpid true"
+      - "--certs-group root"
+  roles:
+    - foreman_installer
+
+- hosts: pipeline-app-nightly-centos7-split
+  become: yes
+  vars:
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--katello-manage-application true"
+      - "--katello-manage-candlepin false"
+      - "--katello-candlepin-hostname pipeline-services-nightly-centos7-split.{{ ansible_domain }}"
+      - "--katello-manage-qpid false"
+      - "--katello-qpid-hostname pipeline-services-nightly-centos7-split.{{ ansible_domain }}"
+      - "--katello-manage-pulp false"
```

Comment:
This should also include `--no-enable-foreman-proxy --no-enable-foreman-proxy-content --no-enable-foreman-proxy-plugin-pulp`. Though I don't think its currently possible to deploy Katello without a integrated foreman-proxy, I'll test it in the next couple of days.

If foreman-proxy is running on one machine (app) and Pulp on another (service) actions like a manifest refresh will fail as `pulp_url` is created using the foreman-proxies hostname.

https://github.com/Katello/katello/blob/master/app/models/katello/ping.rb#L101-L102
https://github.com/Katello/katello/blob/master/app/lib/katello/capsule_content.rb#L134-L136